pages.php

```diff
@@ -32,6 +32,35 @@
     <head>
     </head>
     <body>
-        <?php /* Use PHP's include() method to render the correct page content in here */ ?>
+        <?php /* Use PHP's include() method to render the correct page content in here */ 
+            echo'
+                <a href="/pages.php?page=home"> ++Home++ </a>
+                <a href="/pages.php?page=about"> ++About++ </a>
+                <a href="/pages.php?page=interests"> ++Interests++ </a>';
+                
+                function about (){
+                    include '/pages/about.php';
```

Comment:
The way we do that is by using PHP to tell it the directory the current file is in (that's `pages.php`). 

The current file's location is held in the magic `__FILE__` variable. The function `dirname()` will give us the absolute directory path for any file. So if we do `dirname(__FILE__)` then what we get is the absolute (complete) directory path for `pages.php`.

We then just need to add on the `'/pages/about.php'` bit.